### ISSUE TITLE
Controlling Sonos speakers with Ikea Sound Remote through homebridge/homebridge-zp (question/feature request)

### ISSUE THREAD

**Issue by fraannk** [ebaauw/homebridge-zp#113]
I was really not sure where to ask this question since it's probably not up to you to add this, but if that is not the case, here's my question anyways. 

I bought two Ikea Sound Remotes and a Trådfri hub, and it works great. However, since I always have my speakers grouped to the same big group, and mute and/or adjust volume individually, I don't like that the remote adjusts volume for the entire group. I currently use homebridge-zp to mute/unmute and adjust volume with other homebridge/HomeKit stuff, and since Ikea does not offer a way to change this behaviour, I was wondering if it was possible to adjust volume individually with the remote using homebridge/homebridge-zp? I already do this with a hue Dimmer Switch, so it would "just" be a matter of having the Ikea Sound Remote support HomeKit/homebridge or directly through homebridge-zp. 

TLDR; Is controlling a Sonos **_speaker_** (not group) with the Ikea Sound Remote possible?

**Comment by ebaauw** [comment]
The Symfonisk controller actually sends regular ZigBee commands to control (a group of) ZigBee lights.  The Trådfri hub translates this to Sonos commands.  It’s not possible to use the controller without a ZigBee gateway.

I paired my controller to a deCONZ ZigBee gateway, exposing it as a stateless programmable switch to HomeKit through homebridge-hue.  Assigning the corresponding actions in HomeKit, you can decide for yourself whether to control a single zone or a zone group.
